packages/stream_chat/lib/src/api/channel.dart

```diff
@@ -195,23 +197,22 @@ class Channel {
     String messageId,
     Iterable<String> attachmentIds,
   ) {
-    final message = state.messages.firstWhere(
+    final message = state!.messages!.firstWhereOrNull(
       (it) => it.id == messageId,
-      orElse: () => null,
     );
 
     if (message == null) {
       throw Exception('Error, Message not found');
     }
 
-    final attachments = message.attachments.where((it) {
+    final attachments = message.attachments!.where((it) {
```

Comment:
I think we can set the default value of attachments to an empty list in our Message class, It will greatly reduce these bangs.